submissions/madmaxWMFU/google-popup/css/style.css

```diff
@@ -428,3 +427,12 @@ a>.myAccount {
 hr {
     margin: 20px;
 }
+
+ul {
+    margin: 5px;
+}
+
+ul>li,
+.divLink {
```

Comment:
Variable names should not be too general or too short or too obscure.

Reply:
Done!